auth-api/migrations/versions/2023_08_01_ba5fafee24c1_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
Please reorder the migrations, so we don't need this. Ideally they'd be sequential as they're getting merged into main.

Reply:
Done. 
What I did was:
rollback, manually edited migration files to point which one is before, did upgrade again. Had to clean up the data from the table though. Let me know if it looks good now.

Reply:
Looks great, that's what we usually do - pain in the neck but then the migrations are linear